src/util/audiosignal.cpp

```diff
@@ -4,6 +4,31 @@
 
 namespace mixxx {
 
+// Definitions of static class constants are required by
+// LLVM CLang, but cause "multiply defined symbols" errors
+// in Visual Studio 2013. GCC is happy either with or
+// without those definitions.
+// TODO(XXX): Replace 'static const' with 'static constexpr'
+// in the corresponding .h file after upgrading to Visual
+// Studio 2015 and remove all definitions from this .cpp file.
+#if !defined(_MSC_VER) || (_MSC_VER >= 1900)
```

Comment:
GCC: With or without definitions, doesn't matter -> **optional***
LLVM: Definitions are **mandatory**
Visual Studio 2013: Definitions produce link errors -> **forbidden**

There is no intersection that works for all 3 compilers: I decided to add the definitions as a default with an exception for Visual Studio <= 2013. Starting with the Visual Studio 2015 the definitions will be included (as for GCC/LLVM) and we will see what happens.